index.ts

```diff
@@ -71,13 +71,6 @@ const program = new Commander.Command(packageJson.name)
     `
 
   Select a template to bootstrap the application with.
-`,
```

Comment:
If we remove `engine` so there is no flag for simple chat, then the cli always asks for the data source. I think we can add a `--no-data-source` option for that case

Reply:
ah that's why my tests failed! thanks